### PR TITLE
Add number validation

### PR DESCRIPTION
**Pull request by ruslanfursa**

### CODE REVIEW

File: src/io/smth/validation/SurnameValidator.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        if (!isNullEmptyLength(s)) {
+            return false;
+        }
+        s = s.trim();
+        return isOnlyLetters(s);
+
+    }
+
+    private boolean isNullEmptyLength(String s) {
+        if (s == null || s.length() < 1 || s.length() > 15) {
```
Comment (YahorHr) [inline src/io/smth/validation/SurnameValidator.java]: replace magic numbers to constants

File: src/io/smth/validation/MailValidator.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+public class MailValidator implements Validator{
+    @Override
+    public boolean validate(String s) {
+       if(!isNullEmptyLength(s)){
+           return false;
+       }
+        return AreAllSignsCorrect(s);
+    }
+    private boolean isNullEmptyLength(String s){
+        if(s == null || s.length() < 8 || s.length() > 25){
```
Comment (YahorHr) [inline src/io/smth/validation/MailValidator.java]: replace magic numbers to constants

File: src/io/smth/validation/SurnameValidator.java (hunk 1 above)
Comment (ruslanfursa) [reply src/io/smth/validation/SurnameValidator.java]: done

File: src/io/smth/validation/MailValidator.java (hunk 2 above)
Comment (ruslanfursa) [reply src/io/smth/validation/MailValidator.java]: done

**Review by YahorHr — APPROVED** [review]

**Pull request merged**